test/services/helpers/services/classes.js

```diff
@@ -27,8 +26,72 @@ const cleanup = app => () => {
 	return ids.map(id => app.service('classes').remove(id));
 };
 
+const createByName = app => async ([gradeLevelName, className, schoolId]) => {
+	const gradeLevels = await app.service('gradeLevels').find({
```

Comment:
use findOne instead of find here?

Reply:
Feathers services don't implement `findOne`